sentry/src/routes.rs

```diff
@@ -76,9 +76,16 @@
 //!
 //! Response: [`LastApprovedResponse`][primitives::sentry::LastApprovedResponse]
 //!
-//! - `POST /v5/channel/:id/pay` (auth required)
+//! - [`POST /v5/channel/:id/pay`](channel::channel_payout)
 //!
-//! TODO: implement and document as part of issue #382
+//! This route handles withdrawals of advertiser funds for a spender. Has authentication of the spender.
```

Comment:
it can handle all withdraws, not only advertiser.
Can you rephrase the sentence?

I think we can even say `for the authenticated spender` and drop the second sentence.